### PR TITLE
feature/internships filter

### PR DESCRIPTION
**Pull request by beliavskaja**
- added filter popover with different selects and multi-selects  and filter button
- added values in mocks
- added filter title
Screenshot 2021-11-11 at 18 05 16
Screenshot 2021-11-11 at 18 05 27
Screenshot 2021-11-11 at 18 35 32

### CODE REVIEW

File: src/components/internshipsFilter/InternshipsFilter.jsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? 'simple-popover' : undefined;
```
Comment (nsukhotskaya) [inline src/components/internshipsFilter/InternshipsFilter.jsx:31]: Simplify please

File: src/components/internshipsFilter/InternshipsFilter.jsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+          <FormControl size="small" fullWidth>
+            <InputLabel>
+              {getFieldLabel('internships.filter.label.language')}
+            </InputLabel>
+            <Select
+              value={filterLanguage}
+              label="Language"
+              onChange={(event) => setFilterLanguage(event.target.value)}
+            >
+              <MenuItem value="English">English</MenuItem>
```
Comment (nsukhotskaya) [inline src/components/internshipsFilter/InternshipsFilter.jsx:93]: Use getFieldLabel please

File: src/components/internshipsFilter/InternshipsFilter.jsx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+          <FormControl size="small" fullWidth>
+            <InputLabel>
+              {getFieldLabel('internships.filter.label.status')}
+            </InputLabel>
+            <Select
+              value={filterStatus}
+              label="Status"
+              onChange={(event) => setFilterStatus(event.target.value)}
+            >
+              <MenuItem value="Open">Open</MenuItem>
```
Comment (nsukhotskaya) [inline src/components/internshipsFilter/InternshipsFilter.jsx:106]: same. suggestion:
common.open

File: src/components/internshipsFilter/InternshipsFilter.jsx [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+              {hrs.map((hr) => (
+                <MenuItem key={hr} value={hr}>
+                  <Checkbox checked={filterMentors.indexOf(hr) > -1} />
+                  <ListItemText primary={hr} />
+                </MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+          <Button size="small" variant="contained">
+            Filter
```
Comment (nsukhotskaya) [inline src/components/internshipsFilter/InternshipsFilter.jsx:187]: common.filter

File: src/pages/internships/Internships.jsx [hunk 5]
```diff
@@ -11,10 +11,10 @@ import {
   Input,
 } from '@mui/material';
 import ManageSearchIcon from '@mui/icons-material/ManageSearch';
-import FilterListIcon from '@mui/icons-material/FilterList';
+// import FilterListIcon from '@mui/icons-material/FilterList';
```
Comment (nsukhotskaya) [inline src/pages/internships/Internships.jsx:14]: Please delete unused comment

**Review by nsukhotskaya — APPROVED** [review]

**Review by OstroumovAlex — APPROVED** [review]

**Pull request merged**